Formula/manticoresearch.rb

```diff
@@ -2,7 +2,7 @@ class Manticoresearch < Formula
   desc "Open source text search engine"
   homepage "https://www.manticoresearch.com"
   url "https://github.com/manticoresoftware/manticoresearch/archive/refs/tags/5.0.3.tar.gz"
```

Comment:
Let's add a comment above the `url` to add a `livecheck` with next stable release so that we can avoid development versions.